src/main/java/com/thebois/models/world/World.java

```diff
@@ -262,7 +307,7 @@ public ITile getTileAt(final Position position) {
 
     @Override
     public ITile getTileAt(final int posX, final int posY) {
-        if (posX < 0 || posY < 0 || posX >= worldSize || posY >= worldSize) {
+        if (posX < 0 || posY < 0 || posX > worldSize || posY > worldSize) {
```

Comment:
Since these positions are zero-indexed, one of them being equal to `worldSize` would mean that they are out of bounds. Why remove the `>=` checks?

Reply:
If I try to get position (50, 50) in a world of size 50, then that would be out of bounds. (49, 49), however, is not.

Reply:
Alright 👌 

Try not to commit bug-finding code in the future!